diff -Naur mozilla-release.orig/browser/base/Makefile.in mozilla-release/browser/base/Makefile.in
--- mozilla-release.orig/browser/base/Makefile.in	2012-06-01 14:03:47.000000000 +0200
+++ mozilla-release/browser/base/Makefile.in	2012-06-06 13:35:49.192000002 +0200
@@ -53,7 +53,7 @@
 
 include $(topsrcdir)/config/rules.mk
 
-PRE_RELEASE_SUFFIX := ""
+PRE_RELEASE_SUFFIX :=
 
 DEFINES += \
 	-DMOZ_APP_VERSION=$(MOZ_APP_VERSION) \
